code/game/objects/items/radio/headset.dm

```diff
@@ -304,10 +304,13 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 			handle_interface(master.loc)
 
 
-/obj/item/radio/headset/mainship/mt
-	name = "engineering radio headset"
+/obj/item/radio/headset/mainship/st
+	name = "technician radio headset"
 	icon_state = "eng_headset"
-	keyslot = new /obj/item/encryptionkey/engi
+/obj/item/radio/headset/mainship/st/Initialize()
+	. = ..()
+	keyslot = new /obj/item/encryptionkey/engi(src)
```

Comment:
You'll have to either do this like the other headsets are done, or to make a generic way to handle those in the base headset Init, the second option is preferable, though not required